hack/make/manifests/crd.mk

```diff
@@ -1,6 +1,14 @@
 -include config.mk
 -include ../prerequisites.mk
 
+manifests/version:
```

Comment:
That target changes helm charts, it doesn't directly influence the manifests and should therefore have a different target